### DISCUSSION TITLE
How can I delete one OS and make the partition available?

### DISCUSSION THREAD

**Discussion by soldatino13** [procount/pinn#828] [Q&A]
I have a Raspberry Pi 4 Model B Rev 1.4 with PINN and 2 OSs on it: LibreELEC and batocera.
How can I delete LibreELEC and make the storage space (partition) available to the other OS (batocera)?

**Comment by procount** [comment]
Are you using an SD card for this?
What capacity does it have?
Do you have a spare SD card, or one with another OS on it, like Raspberry Pi OS? 
Do you have a USB SD card adaptor?
_(ADDED:)_ Do you have a USB memory stick?
Can you post the contents of installed_os.json? It will be on the settings partition 5 (/dev/mmcblk0p5 if using an SD card)
